manual.md

```diff
@@ -1,166 +1,139 @@
-Temperature Check to start a community discussion around an idea, question, or proposal.
+### Off-chain
 
-To start a **Temperature Check**:
+In addition to the on-chain governance system, the RadicleDAO manages a set of off-chain processes and resources that define the rules for participating in the DAO. These processes and resources are maintained off-chain, but updates to these documents still require formal community approval via Snapshot vote.
 
-- Create a topic and label it as follows: “Temperature Check :fire: - Your Topic of Discussion Here”.
-- Ask a **general, non-biased question about a potential change, improvement, or action** (i.e. Should swapping on the Balancer LBP be unpaused?).
-- Add any additional information to contextualize the question for the greater community.
-- Collect feedback and engage in discussion for 3-5 days.
+See Proposal Process for more info.
 
-> It’s important to remain as objective as a view as possible. Present developed arguments and the “full picture” — not
-> just your opinion!
+## **Governance Toolkit**
 
-### Phase 2: Discussion :seedling: - Discourse
+The primary tools for The RadicleDAO governance are:
 
-To move a **Temperature Check** forward, it should then be presented for formal, structured discussion in Phase 2. Phase
-2 requires two things: a **formal discussion** on Discourse and a **drafted proposal** that presents a thorough analysis
-for formal review by Radicle stakeholders.
+- **[Snapshot](https://snapshot.org/#/opcollective.eth)**: Where all Formal Reviews are submitted for vote.
+- **[radicle.community](https://radicle.community/):** A community forum for discussion about governance proposals and topics.
+- **[Discord](https://discord-gateway.optimism.io/)**: For governance announcements and informal governance discussion.
 
-#### Drafted Proposal
+Additional tools for voting and proposal making can be found here.
 
-The goal of this phase is to take the the ideas, arguments, and feedback generated in the Temperature Check and craft an
-overview of the proposal that meets the following criteria:
+## Proposal Process
 
-- **Functional description** (what is this being proposed?)
-- **Purpose** (what’s the “why”?)
-- **Background** (what is the reasoning behind the proposal?)
-- **Link to Temperature Check**
-- **Reasoning & analysis** (what is the case for the proposal? what are the pros and cons?)
-- **Technical implementation** (who will be writing the code? what is the scope required?)
-- **Impact** (how does this contribute to the long-term resilience, sustainability and/or growth of the Radicle network?)
-- **Open questions** (what else needs to be figured out?)
+A Radicle Governance Proposal (RGP) can be proposed on-chain by anyone who has an amount of $RAD greater than or **equal to 1% of total total supply delegated** to their address. RGPs must be formally labeled in the following format: e.g. `RGP-1`
```

Comment:
Add full example of what a proposal title should look like i.e. include proposal type, # and title: [Executeable][RGP - #] - [PROPOSAL TITLE]

Reply:
added in https://github.com/radicle-dev/radicle-governance/pull/2/commits/64fd6363069b0acac9947b6157baaba7a09b4fb2